hs_core/templates/pages/create-resource.html

```diff
@@ -45,11 +45,15 @@
             </div>
 
             <div class="col-xs-12">
-                <h2 class="page-title">Create Resource</h2>
-            </div>
-
-            <div class="col-xs-12 height-fix">
-                <span class="btn btn-primary btn-create-resource pull-right">Create Resource</span>
+                <h2 class="page-title">Create Resource
+                    <small class="text-muted">
+                        <em>DOI is reserved for the resource. </em>
+                        <span class="glyphicon glyphicon-info-sign info-popover-glypn"
+                              data-toggle="popover1" data-placement="bottom"

```

Comment:
Add `data-trigger="hover"` to change the trigger to hover which will allow the popup to close. This behavior also works in mobile devices by tapping on the icon (or outside of it to close).

Reply:
With the hovering I found the popover quickly dismissed once the mouse is outside of the glyph icon. Then I have to move mouse back to the tiny icon to see the content. I prefer the current behavior: click icon->popover shows and stay->user click outside of icon to force it disappear

Reply:
@engineSound the popup does not go away when clicking outside of it. Is that the intended behavior?

Reply:
It is an intended behavior. It is an attention-holding pattern, leading user to read the DOI message in the popover because DOI is an essential offer from hydroshare.